new-site/app/layouts/SidebarLayout.js

```diff
@@ -49,41 +42,35 @@ function Nav({ nav, children, fallbackHref }) {
     <nav
       id="nav"
       ref={scrollRef}
-      className="px-1 pt-6 overflow-y-auto font-medium text-base sm:px-3 xl:px-5 lg:text-sm pb-10 lg:pt-10 lg:pb-16 sticky?lg:h-(screen-18)"
+      className="px-1 mt-6 font-medium text-base sm:px-3 xl:px-5 pb-10 lg:pb-16 sticky?lg:h-(screen-18) overflow-y-auto"
     >
       <ul>
-        <TopLevelNav />
         {children}
         {nav &&
-          Object.keys(nav)
+          nav
             .map((category) => {
-              let publishedItems = nav[category].filter((item) => item.published !== false)
+              let publishedItems = category.pages.filter((item) => item.published !== false)
               if (publishedItems.length === 0 && !fallbackHref) return null
               return (
-                <li key={category} className="mt-8">
-                  <h5
-                    className={clsx(
-                      "px-3 mb-3 lg:mb-3 uppercase tracking-wide font-semibold text-sm lg:text-xs",
-                      {
-                        "text-gray-900": publishedItems.length > 0,
-                        "text-gray-400": publishedItems.length === 0,
-                      }
-                    )}
-                  >
-                    {category}
-                  </h5>
+                <li key={category.title} className="my-5">
+                  {category.title}
                   <ul>
-                    {(fallbackHref ? nav[category] : publishedItems).map((item, i) => (
-                      <NavItem
-                        key={i}
-                        href={item.href}
-                        isActive={item.href === router.pathname}
-                        ref={item.href === router.pathname ? activeItemRef : undefined}
-                        isPublished={item.published !== false}
-                        fallbackHref={fallbackHref}
-                      >
-                        {item.shortTitle || item.title}
-                      </NavItem>
+                    {(fallbackHref ? category.pages : publishedItems).map((item, i) => (
+                      <>
```

Comment:
```suggestion
                      <Fragment key={i}>
```
Key needs to be on fragment